### DISCUSSION TITLE
Thanks for this tool!

### DISCUSSION THREAD

**Discussion by katse007** [AAndyProgram/SCrawler#64] [General]
Hello! Thanks for making this tool! It works great and has a TON of useful functions. 

I would like to ask, any plans/ideas on getting this to work on Windows 7 at all? Even if it loses certain functions like notifications and such?

**Comment by AAndyProgram** [comment]
I don't think so. I disabled Win7 and Win8 compatibility a few releases ago. New notification functions require at least Win10. Of course, I can do it, but I have no desire. Besides, you are the first who asked to return compatibility with Win7. It takes time to develop. Most users have no time to put likes on [another site](https://alternativeto.net/software/scrawler/about/) (which is what I asked for in every release). Even on this site under release, users have no desire to set a like. So I have no desire to develop additional compatibility components (not used by me) for users.

Sorry.
